src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -9363,50 +9365,54 @@ s32 Player_ActionHandler_3(Player* this, PlayState* play) {
+        s16 playerVelYaw = Math_Atan2S_XY(this->actor.velocity.z, this->actor.velocity.x);
+        Vec3f slopeNormal;
+        s16 downwardSlopeYaw;
+        s16 velYawToDownwardSlope;
+        f32 slopeSlowdownSpeed;
+        f32 temp_fv1;
+        f32 var_fa1;
+        f32 slopeSlowdownSpeedStep;
 
-            Actor_GetSlopeDirection(this->actor.floorPoly, &slopeNormal, &downwardSlopeYaw);
-            velYawToDownwardSlope = downwardSlopeYaw - playerVelYaw;
+        Actor_GetSlopeDirection(this->actor.floorPoly, &slopeNormal, &downwardSlopeYaw);
+        velYawToDownwardSlope = downwardSlopeYaw - playerVelYaw;
 
-            if (ABS_ALT(velYawToDownwardSlope) > 0x3E80) { // 87.9 degrees
-                var_fa1 = (Player_Action_96 == this->actionFunc) ? Math_CosS(this->floorPitch) : slopeNormal.y;
-                slopeSlowdownSpeed = (1.0f - var_fa1) * 40.0f;
-                temp_fv1 = fabsf(this->actor.speed) + slopeSlowdownSpeed;
-                slopeSlowdownSpeedStep = SQ(temp_fv1) * 0.011f;
-                slopeSlowdownSpeedStep = CLAMP_MIN(slopeSlowdownSpeedStep, 2.2f);
+        if (ABS_ALT(velYawToDownwardSlope) > 0x3E80) { // 87.9 degrees
+            var_fa1 = (Player_Action_96 == this->actionFunc) ? Math_CosS(this->floorPitch) : slopeNormal.y;
+            slopeSlowdownSpeed = (1.0f - var_fa1) * 40.0f;
+            temp_fv1 = fabsf(this->actor.speed) + slopeSlowdownSpeed;
+            slopeSlowdownSpeedStep = SQ(temp_fv1) * 0.011f;
+            slopeSlowdownSpeedStep = CLAMP_MIN(slopeSlowdownSpeedStep, 2.2f);
 
-                // slows down speed as player is climbing a slope
-                this->pushedYaw = downwardSlopeYaw;
-                Math_StepToF(&this->pushedSpeed, slopeSlowdownSpeed, slopeSlowdownSpeedStep);
-            } else {
-                Player_SetAction(play, this, Player_Action_73, 0);
-                func_8082DE50(play, this);
-                Player_Anim_PlayLoopMorph(play, this, sPlayerSlopeSlipAnims[this->av1.actionVar1]);
-                this->speedXZ = sqrtf(SQXZ(this->actor.velocity));
-                this->yaw = downwardSlopeYaw;
-                if (sPlayerFloorPitchShape >= 0) {
-                    this->av1.actionVar1 = 1;
-                    Player_AnimSfx_PlayVoice(this, NA_SE_VO_LI_HANG);
-                }
+            // slows down speed as player is climbing a slope
+            this->pushedYaw = downwardSlopeYaw;
+            Math_StepToF(&this->pushedSpeed, slopeSlowdownSpeed, slopeSlowdownSpeedStep);
+        } else {
+            // moving downward on the slope, causing player to slip and then slide down
+            Player_SetAction(play, this, Player_Action_SlideOnSlope, 0);
+            func_8082DE50(play, this);
 
-                return true;
+            // facingUpSlope has not yet been updated based on slope, so it will always be 0 here.
+            Player_Anim_PlayLoopMorph(play, this, sSlopeSlideAnims[this->av1.facingUpSlope]);
```

Comment:
Just want to comment that `facingUpSlope` in OoT was updated before this animation was set. However in MM, they updated `facingUpSlope` below meaning `facingUpSlope` is always 0 as it's reset to 0 in `Player_SetAction` and not updated in time for this animation